docs-gen/content/releases.md

```diff
@@ -8,6 +8,10 @@ weight: 2
 
 {{< directoryindex path="/static/releases/nightly" pathURL="/vehicle_signal_specification/releases/nightly/" >}}
 
+### Version 2.1
+
+{{< directoryindex path="/static/releases/v2.0" pathURL="/vehicle_signal_specification/releases/v2.1/" >}}
```

Comment:
My apologies, I did not review thoroughly last time, we have a "v2.0" on this line as well in path

Reply:
I mean, I made the mistake :) I'll push an update